src/tests/ftest/util/MpioUtils.py

```diff
@@ -88,3 +88,50 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
 
         except (IOError, OSError, paramiko.SSHException, socket.error) as e:
             raise MpioFailed("<ROMIO Test FAILED> \nException occurred: {}".format(str(e)))
+
+    def run_ior(self, basepath, pool_uuid, svcl, np, hostfile, ior_flags, iteration,
```

Comment:
Do we really want separate function or may be better to maintain the one which is under IorUtils.py and pass the parameter and protocol [MPIIO/DAOS/POSIX], otherwise we need to maintain multiple 3-4 files for same IOR run

Reply:
Done.